src/JackTrip.cpp

```diff
@@ -1050,13 +1050,17 @@ void JackTrip::stop(const QString& errorMessage)
     mHasShutdown = true;
     std::cout << "Stopping JackTrip..." << std::endl;
 
-    // Stop The Sender
-    mDataProtocolSender->stop();
-    mDataProtocolSender->wait();
+    if (mDataProtocolSender != nullptr) {
```

Comment:
Is there a specific reason you're removing these checks?

Reply:
I'm not. I moved them under the protection of that if. If those dataprotocol objects are null (like when we fail out of setting up audio for sample rate reasons), we get a segfault trying to shut down the jacktrip process.